apps/examples/wifi_manager/wm_test/wm_test_usage.h

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2021 Samsung Electronics All Rights Reserved.
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
```

Comment:
Please revert this one too
You need a change of the year only when the file added is new

Reply:
Done